dem/cli/main.py

```diff
@@ -207,6 +207,18 @@ def delete(dev_env_name: Annotated[str, typer.Argument(help="Name of the Develop
     else:
         raise InternalError("Error: The platform hasn't been initialized properly!")
 
+@typer_cli.command()
+def install(dev_env_name: Annotated[str, typer.Argument(help="Name of the Development Environment to install.",
+                                                       autocompletion=autocomplete_dev_env_name)]) -> None:
+    """
+    Install the Development Environment from the local setup. If a tool image is not installed
+    by any of the available local Development Environments, the DEM will install them.
```

Comment:
` If a tool image is not installed
    by any of the available local Development Environments, the DEM will install them.`
  IMO this part of the comment is not really helpful for the user, so delete it, please.